packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -208,11 +208,15 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 
 	let delayTimeout: number;
 	const delayTime = 333;
+
 	const valueProps = {
-		onMouseEnter: (e: React.MouseEvent<HTMLAnchorElement>) => {
+		onMouseEnter: (e: React.MouseEvent<HTMLAnchorElement>, activeTerm: any) => {
 			clearTimeout(delayTimeout);
 			delayTimeout = window.setTimeout(() => {
 				(e.target as HTMLAnchorElement).focus();
+				if (activeTerm) {
```

Comment:
activeTerm typing is loose, may want to check for preview method before trying to run it.

Reply:
Should have typing for activeTerm perhaps?

Reply:
To be revisted...